packages/common/src/interfaces.ts

```diff
@@ -175,6 +175,7 @@ export interface VerkleAccessWitnessInterface {
  *
  */
 export interface StateManagerInterface {
+  name?: string
```

Comment:
I know this was suggested in the issue, but I'm not sure if I like the property name `name`, it is very general. Something like `statemanagerType` or `smType` would also work. I also think this should be mandatory, so this cannot be `undefined`.